Source/ThirdParty/libwebrtc/Configurations/vp9_encoder_fuzzer.xcconfig

```diff
@@ -26,11 +26,13 @@
 PRODUCT_NAME = vp9_encoder_fuzzer;
 
 CODE_SIGN_STYLE = Automatic;
-ENABLE_LIBFUZZER = YES;
 
 HEADER_SEARCH_PATHS = $(inherited) Source/webrtc Source/third_party/abseil-cpp;
 
 OTHER_CFLAGS = $(inherited) -DENCODER=vp9;
 OTHER_LDFLAGS = $(inherited) -lwebrtc;
 
 WARNING_CFLAGS = $(inherited) -Wno-constant-logical-operand;
+
```

Comment:
Do we need to add these lines here?

```suggestion

EXCLUDED_SOURCE_FILE_NAMES = $(EXCLUDED_SOURCE_FILE_NAMES_$(ENABLE_LIBFUZZER));
EXCLUDED_SOURCE_FILE_NAMES_ = *;
EXCLUDED_SOURCE_FILE_NAMES_YES = ;

```

Reply:
We do, this is an oversight. (actually somewhat surprised this built in EWS without these lines)